.editorconfig

```diff
@@ -0,0 +1,140 @@
+root = true
+
+[*]
+charset = utf-8
+indent_size = 4
```

Comment:
Why is this set to `4`? We have `2` everywhere. This messes up the IDE.

Reply:
Sorry, fixed it here:
https://github.com/airbytehq/airbyte/pull/9235

Reply:
Cool. Thank you.